lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -56,4 +55,7 @@
   <ItemGroup>
     <EmbeddedResource Include="Injected\injected.js" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
should this be a PackageReference to https://github.com/webdriverbidi-net/webdriverbidi-net ?

Reply:
I'm just backing up my work when I switch between branches.
We will use the real package.